WordPress/WordPressTest/Test Data/TestPlans.swift

```diff
@@ -43,12 +43,17 @@ enum TestPlans {
         }
     }
 
-    static let allPlans = {
-        return [
-            TestPlans.free,
-            TestPlans.premium,
-            TestPlans.business
-            ]
-            .map({ $0.plan })
-    }()
+    var product: MockProduct {
+        let plan = self.plan
```

Comment:
Ah, I didn't realize `plan` was a computed variable. Even then, I think it's best to remove it as it looks confusing and relies on an implementation detail. If performance was an issue, I'd turn `plan` into a `lazy var` instead.

This bit from the [Swift API Guidelines](https://swift.org/documentation/api-design-guidelines/#general-conventions) seems relevant:

> **Document the complexity of any computed property that is not O(1).** People often assume that property access involves no significant computation, because they have stored properties as a mental model. Be sure to alert them when that assumption may be violated.

Although this being mock data for a unit test, I don't think it really matters much 😄